bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/metrics/BigtableClientMetrics.java

```diff
@@ -164,6 +174,31 @@ public void close() {
     }
   }
 
+  /**
+   * Creates a named {@link Meter}.
+   *
+   * @param name
+   * @return a Dropwizard Metrics {@link com.codahale.metrics.Meter} or {@link BigtableClientMetrics#NULL_METER}.
```

Comment:
+1 on removing the leakiness.  Currently, all 3 types of metrics have comments leak implementation details.  I plan to a MetricsFactory interface, with Null and Dropwizard implementations.  Is it ok to fix all of the comments with the next PR?